test_analysis/fake_data_stan_analysis_termite.py

```diff
@@ -0,0 +1,513 @@
+#
+# fake_data_stan_analysis.py
+# Author: T. E. Weiss
+# Date modified: June 2, 2020
+#
+# This script generates fake data, then analyzes the data in Stan to infer posteriors.
+# Pathnames configured for running from: termite/phase2_main_scripts/
+#
+
+"""
```

Comment:
are these to-do's done? please update comment to current status and plan.

Reply:
Good question. Regarding each to-do:

• "Tracking of convergence issues" – that isn't accounted for in FakeExperimentEnsemble; it has to be done separately, after running. This function, or one like it, could still be updated to include convergence-issue tracking, e.g. for a Bayesian analysis for Phase III. So, I might actually leave this to-do.

• "An option to parallelize with slurm instead of multiprocessing" – Also not implemented, but I did run with slurm, by using a different script to parallelize multiple instances of FakeExperimentEnsemble. That worked fine, so I can delete this to-do.

• "Run morpho processor for ensemble-analysis plotting, once it has been tested sufficiently" – this was done outside of morpho, but I wanted to implement it in morpho. This is a morpho to-do as opposed to a mermithid to-do, so I'll delete it.